.github/actions/generic-ci/action.yml

```diff
@@ -256,11 +256,18 @@ runs:
       working-directory: ${{github.workspace}}
       run: ctest --test-dir build_examples -C Release -VV
 
+    - name: Install optional component f3d plugins
+      shell: bash
+      working-directory: ${{github.workspace}}/build
+      run: cmake --install . --component plugin_sdk
+
+      # CMAKE_MODULE_PATH is required because of
+      # https://github.com/AcademySoftwareFoundation/openvdb/issues/1160
     - name: Build plugin examples
       shell: bash
       working-directory: ${{github.workspace}}
       run: |
-        cmake -Werror=dev -Werror=deprecated --warn-uninitialized -Df3d_DIR:PATH=$(pwd)/install/lib/cmake/f3d -DCMAKE_PREFIX_PATH:PATH=$(pwd)/dependencies/install/ -DCMAKE_INSTALL_PREFIX=$(pwd)/install/ -DBUILD_TESTING=ON -DCMAKE_OSX_ARCHITECTURES=${{ inputs.cpu }} -B build_plugins source/examples/plugins
+        cmake -Werror=dev -Werror=deprecated --warn-uninitialized -DCMAKE_MODULE_PATH=$(pwd)/dependencies/install/lib/cmake/OpenVDB/ -Df3d_DIR=$(pwd)/install/lib/cmake/f3d -Df3dPlugin_DIR:PATH=$(pwd)/install/lib/cmake/f3dPlugin -DCMAKE_PREFIX_PATH:PATH=$(pwd)/dependencies/install/ -DCMAKE_INSTALL_PREFIX=$(pwd)/install/ -DBUILD_TESTING=ON -DCMAKE_OSX_ARCHITECTURES=${{ inputs.cpu }} -B build_plugins source/examples/plugins
```

Comment:
Why is there a change related to OpenVDB?

Reply:
```
 # CMAKE_MODULE_PATH is required because of
 # https://github.com/AcademySoftwareFoundation/openvdb/issues/1160
```

Not sure why it was not needed before though